### PR TITLE
conf: Add nginx config

### PR DESCRIPTION
**Pull request by punfil**
Add support for PHP pages.

### CODE REVIEW

File: config/nginx/default [hunk 1]
```diff
@@ -0,0 +1,23 @@
+server {
+	listen 80 default_server;
+	listen [::]:80 default_server;
+
+
```
Comment (bartoszLesniewski) [inline config/nginx/default]: Too many new lines.

File: config/nginx/default [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	}
+
+	location ~ \.php$ {
+		include snippets/fastcgi-php.conf;
+		fastcgi_pass unix:/run/php/php8.2-fpm.sock;
+	}
+
+}
+
+
```
Comment (bartoszLesniewski) [inline config/nginx/default]: Too many new lines.

File: config/nginx/default [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	server_name _;
+
+	location / {
+		try_files $uri $uri/ =404;
+	}
+
+	location ~ \.php$ {
+		include snippets/fastcgi-php.conf;
+		fastcgi_pass unix:/run/php/php8.2-fpm.sock;
```
Comment (bartoszLesniewski) [inline config/nginx/default]: If php in 8.2 version will be installed on the device, it's ok, but I had to change it when testing on VM :P

File: config/nginx/README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+## Where to put the configuration files?
+- default -> /etc/nginx/sites-available/default
+
+## Other actions required
+None
```
Comment (bartoszLesniewski) [inline config/nginx/README.md]: What about changing persmissions and ownership to the website directory (/var/www/html)? I had "403 Forbidden" error without this:
`chmod -R 775 /var/www/html/`
`chown -R www-data:www-data /var/www/html/`
Does it work without it on your machine?

**Review by bartoszLesniewski — CHANGES_REQUESTED** [review]

File: config/nginx/default (hunk 3 above)
Comment (punfil) [reply config/nginx/default]: @esesyn01 FYI, this value should be adjusted to version installed on the image. Please let us know which version will be installed there, preferably the latest, so we can edit it before merging.

File: config/nginx/README.md (hunk 4 above)
Comment (punfil) [reply config/nginx/README.md]: I did it a bit differently, but you're right that it should be applied.
Done

File: config/nginx/default (hunk 1 above)
Comment (punfil) [reply config/nginx/default]: Done

File: config/nginx/default (hunk 2 above)
Comment (punfil) [reply config/nginx/default]: Done

**Review by bartoszLesniewski — APPROVED** [review]
No further findings. Approved.

File: config/nginx/default (hunk 3 above)
Comment (esesyn01) [reply config/nginx/default]: @punfil Supported php version in meta-oe layer (langdale) is `8.1.16`, so if it generates any trouble, change what is needed

Comment (punfil) [reply config/nginx/default]: Changed PHP version to 8.1.16.

**Review by esesyn01 — APPROVED** [review]
LGTM

**Pull request merged**